### ISSUE TITLE
Sandboxes are no longer supported by Codesandbox CI?

### ISSUE THREAD

**Issue by wojciechczerniak** [codesandbox/codesandbox-client#3541]
# ❔ Question

Hi,

What happend to Codesandbox CI? The bot is removing sandboxes from the comments:

![obraz](https://user-images.githubusercontent.com/4718998/74763073-5f373800-527f-11ea-8aa5-7a20a267694c.png)

Is it a bug o configuration file format has changed?

**Comment by garethx** [comment]
Link to PR for context: https://github.com/facebook/react/pull/18064

**Comment by lbogdan** [comment]
Hey @wojciechczerniak ,

Thanks for reporting this issue! What happened is that the CodeSandbox API threw an error when we forked the sandbox, which we currently just ignore, so the build ended up having no associated sandbox(es). We should probably retry API calls on recoverable HTTP error codes (now we got `429 Too Many Requests`, which was most probably returned by CloudFlare). I'll get back to you whenever we figure out a fix for this.
